src/Tools/Source/RunTests/ProcDumpUtil.cs

```diff
@@ -86,30 +86,25 @@ internal static class ProcDumpUtil
     {
         internal static Process AttachProcDump(ProcDumpInfo procDumpInfo, int processId)
         {
-            return AttachProcDump(procDumpInfo.ProcDumpFilePath, processId, procDumpInfo.DumpDirectory);
+            // -ma	Write a 'Full' dump file. Includes All the Image, Mapped and Private memory.
+            // -e	Write a dump when the process encounters an unhandled exception. Include the 1 to create dump on first chance exceptions.
+            // -f C00000FD.STACK_OVERFLOW Dump when a stack overflow first chance exception is encountered. 
+            return StartProcDump(procDumpInfo, processId, "-ma -e -f C00000FD.STACK_OVERFLOW");
```

Comment:
💭 I'm guessing the changes in this file aren't necessary for the simplified change we would merge?

Reply:
I'm going to keep this part so at least any future need of procdump (even as a temporary investigation tool) still has this available.

Reply:
On second thought keeping this PR short, will revisit this later since there's other cleanup needed too.